python/lsst/sconsUtils/state.py

```diff
@@ -215,6 +215,14 @@ def _initEnvironment():
       CC_LOGGER_BIN
     """.split()
 
+    implicitMultithreadingVars = """
+      OMP_NUM_THREADS
+      MKL_NUM_THREADS
+      OPENBLAS_NUM_THREADS
+      NUMEXPR_MAX_THREADS
+      NUMEXPR_NUM_THREADS
+    """.split()
```

Comment:
Why this way instead of making a list directly?

Reply:
I'm following the code above ... when in Rome.

Reply:
Why the heck were they done that way?